### DISCUSSION TITLE
Is it possible to remove google fonts dependency?

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#10477] [Q&A] [answered]
### Description

I don't know why but I often get the problem of quarto not being able to access Google Fonts:

```
[WARNING] Could not fetch resource https://fonts.googleapis.com/css2?
family=Source+Sans+Pro:wght@300;400;700&display=swap: HttpExceptionRequest Request {
    host                 = "fonts.googleapis.com"
    port                 = 443
    secure               = True
    requestHeaders       = []
    path                 = "/css2"
    queryString          = "?family=Source+Sans+Pro:wght@300;400;700&display=swap"
    method               = "GET"
    proxy                = Nothing
    rawBody              = False
    redirectCount        = 10
    responseTimeout      = ResponseTimeoutDefault
    requestVersion       = HTTP/1.1
    proxySecureMode      = ProxySecureWithConnect
  }
   ConnectionTimeout
   ```

Despite I being able to access the same address in the browser: https://fonts.googleapis.com/css2?
family=Source+Sans+Pro:wght@300;400;700&display=swap:

I am creating a quarto manuscript format with html and typst. I tried to add `theme: none` in the _extension.yml format html section. But still got the googleapis dependency.
What can I do?

**Comment by mcanouil** [comment]
Could you share a **small self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? The goal is to make it as easy as possible for us to recreate your problem so that we can fix it: please help us help you! Thanks.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by fredguth** [reply]
Actually, it is happens sometimes only.  I don't understand the conditions.  It works with any project with html format, so there is no mwe in this case. In the same way that it starts breaking... it stops breaking.  I don't believe it is an error in Quarto but I problem with Google.

**Reply by gordonwoodhull** [reply, marked as answer]
maybe see #4387
